lib/teleterm/daemon/daemon.go

```diff
@@ -171,6 +172,10 @@ func (s *Service) CreateGateway(ctx context.Context, params CreateGatewayParams)
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	if gateway, ok := s.findGatewayByTargetURI(params.TargetURI); ok {
```

Comment:
This is fine for kube gateways, but when it comes to dbs, the user can open multiple gateways for a single `TargetURI`, they're actually scoped by the `TargetURI + db username` combo.

For now I'd think we could just make this behavior special for kube gateways and explain why it's done this way.